packages/manager/apps/pci-public-ip/src/pages/list/ListGuard.tsx

```diff
@@ -25,7 +27,11 @@ export default function ListGuard({
 
   useEffect(() => {
     if (!isFailoverIPsLoading && !isFloatingIPsLoading) {
-      if (failoverIPs?.length > 0 || floatingIPs?.length > 0) {
+      if (
+        floatingIpCreation ||
+        failoverIPs?.length > 0 ||
```

Comment:
After the cache cleaning creation, the list should be not empty no ? and no need after that to check the boolean `floatingIpCreation` ?

Reply:
i did try (cache cleaning after creation) but it seems there is some delay before the ip is created